homeassistant/components/freebox/router.py

```diff
@@ -161,10 +161,16 @@ async def _update_disks_sensors(self) -> None:
     async def _update_raids_sensors(self) -> None:
         """Update Freebox raids."""
         # None at first request
-        fbx_raids: list[dict[str, Any]] = await self._api.storage.get_raids() or []
+        try:
+            fbx_raids: list[dict[str, Any]] = await self._api.storage.get_raids() or []
 
-        for fbx_raid in fbx_raids:
-            self.raids[fbx_raid["id"]] = fbx_raid
+            for fbx_raid in fbx_raids:
+                self.raids[fbx_raid["id"]] = fbx_raid
+        except HttpRequestError:
+            _LOGGER.warning("Unable to enumerate raid disks")
```

Comment:
Does the `HttpRequestError` include some information, which would help the user to identify the problem?

Reply:
Unfortunately the message is generic, it is returned: Internal error